layers/+chat/jabber/packages.el

```diff
@@ -15,16 +15,20 @@
   (use-package jabber
     :defer t
     :init (spacemacs/set-leader-keys "aj" 'jabber-connect-all)
-    :config (spacemacs/set-leader-keys-for-major-mode 'jabber-roster-mode
-              "a" 'jabber-send-presence
-              "b" 'jabber-get-browse
-              "d" 'jabber-disconnect
-              "e" 'jabber-roster-edit-action-at-point
-              "g" 'jabber-display-roster
-              "i" 'jabber-get-disco-items
-              "j" 'jabber-muc-join
-              "q" 'bury-buffer
-              "r" 'jabber-roster-toggle-offline-display
-              "s" 'jabber-send-subscription-request
-              "v" 'jabber-get-version
-              "RET" 'jabber-roster-ret-action-at-point)))
+    :config (progn
```

Comment:
The `progn` would better start on the next line to follow the style of the rest of the code :-)